### PR TITLE
✨ Lakefs

### PR DESCRIPTION
**Pull request by gittysachin**
Lakefs onboarding (https://github.com/pluralsh/plural-artifacts/issues/305), initial push for testing

### CODE REVIEW

**Comment by gittysachin** [comment]
@michaeljguarino please let me know if need to modify the terraform files because the actual LakeFS terraform script has to be different with more setup.

File: lakefs/plural/recipes/lakefs-aws.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  name: ingress-nginx-aws
+
+- repo: postgres
+  name: aws-postgres
+
+sections:
+- name: lakefs
+  configuration:
+  - name: hostname
+    documentation: domain on which you'd like to host lakefs's page
```
Comment (michaeljguarino) [inline lakefs/plural/recipes/lakefs-aws.yaml:18]: same as prior

File: lakefs/helm/lakefs/templates/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{{- if .Values.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "lakefs.fullname" . }}
+  labels:
+    {{- include "lakefs.labels" . | nindent 4 }}
+type: Opaque
+data:
+  database_connection_string: {{ .Values.secrets.databaseConnectionString | default "" | b64enc }}
```
Comment (michaeljguarino) [inline lakefs/helm/lakefs/templates/secret.yaml:10]: I'm not sure I see where these are being set currently

**Comment by michaeljguarino** [comment]
my guess is you'll need to add some terraform to create the s3/gcs/etc buckets too, but I'm not sure about lakefs' details, it's possible that's resolved in the application itself

**Comment by michaeljguarino** [comment]
I'll merge this to unblock the chicken-egg for publishing this

**Pull request merged**